uportal-war/src/main/java/org/jasig/portal/rendering/StructureAttributeSource.java

```diff
@@ -30,8 +32,15 @@
  * @version $Revision$
  */
 public class StructureAttributeSource extends StylesheetAttributeSource {
+    private static final Logger LOGGER = LoggerFactory.getLogger(StructureAttributeSource.class);
```

Comment:
Since you are exposing a logger for subclasses with getLogger(), you could probably just have done
protected final Logger LOGGER = LoggerFactory.getLogger(this.getClass());
instead.